### ISSUE TITLE
A11y_SQLAzureDataStudio_TreeComponent_ScreenReader: NVDA is not reading the labels associated with the TreeComponent checkbox while navigating using Tab Key

### ISSUE THREAD

**Issue by nahk-ivanov** [microsoft/azuredatastudio#17672]
- Azure Data Studio Version: 1.33.1 (system setup)
- Commit: 0b9ef714c823d662815f441aec4a94eaa21940f8
- Date: 2021-11-04T17:45:15.872Z
- VS Code: 1.58.0
- Electron: 12.0.9
- Chrome: 89.0.4389.128
- Node.js: 14.16.0
- V8: 8.9.255.25-electron.0
- OS: Windows_NT x64 10.0.22000

**Pre-requisite:**
Build and install sample `sqlservices` extension

**Repro Steps:**
1. Run Azure DevOps Studio Insider.
2. From Command palette, run `sqlservice.openDialog` command and navigate to `Test tab 3`.
3. Using Tab key, navigate one of the checkboxes in the tree:
![image](https://user-images.githubusercontent.com/11728682/141852423-c415c02c-c4de-4e96-a566-eb972deaa267.png)

**Actual:**
On tab navigation to the checkbox, NVDA is reading the checkbox as “checkbox unchecked”. The text associated with the tree node is not announced.

Same issue with down key navigation 

**Expected:**
The NVDA should also read the label associated with the checkbox when navigating to it using tab key

**User Impact:**
The user may not be able to understand properly about which checkbox is visited.

**MAS Reference:**
MAS 4.1.2 – Name, Role, Value
